examples/yaml/service-common/avs/maximal-aerospike-vector-search.yaml

```diff
@@ -0,0 +1,127 @@
+    - "index"
+    - "search"
+
+feature-key-file: "/etc/aerospike/features.conf"
+
+aerospike:
+  seeds:
+    - aerospike-node-1:
+        port: 3000
+    - aerospike-node-2:
+        port: 3001
+  client-policy:
+    credentials:
+      username: "admin"
+      password-file: "/etc/aerospike/password.txt"
+    tls-id: "aerospike-tls"
+    max-conns-per-node: 1000
+
+logging:
+  timezone: "UTC"
+  format: "json"
+  file: "/var/log/aerospike-vector-search.log"
+  levels:
+    metrics-ticker: "debug"
+    com.aerospike.vector.index: "debug"
+    com.aerospike.vector.storage: "info"
+  max-history: 30
+  ticker-interval: 10
+
+heartbeat:
+  seeds:
+    - address: "10.0.0.1"
+      port: 5001
+    - address: "10.0.0.2"
+      port: 5001
+
+interconnect:
+  client-tls-id: "interconnect-tls"
+  ports:
+    "5001":
+      addresses:
+        - "127.0.0.1"
+        - "10.0.0.1"
+      tls-id: "interconnect-tls"
+
+security:
+  auth-token:
+    private-key: "/etc/aerospike/private.pem"
+    private-key-password: "secretpassword"
+    public-key: "/etc/aerospike/public.pem"
+    token-expiry: 3600
+
+service:
+  metadata-namespace: "aerospike-meta"
+  ports:
+    "5000":
+      tls-id: "service-tls"
+      advertised-listeners:
+        external:
+          - address: "vector-search.example.com"
+            port: 5443
+
+tls:
+  service-tls:
+    mutual-auth: true
+    allowed-peer-names:
+      - "svc.aerospike.com"
+    trust-store:
+      store-type: "PEM"
+      certificate-files: "/etc/aerospike/tls/ca.pem"
+    key-store:
+      store-type: "PEM"
+      store-file: "/etc/aerospike/tls/service.key.pem"
+      store-password-file: "/etc/aerospike/tls/storepass"
+      certificate-chain-files: "/etc/aerospike/tls/service.crt.pem"
+
+  interconnect-tls:
+    mutual-auth: true
+    allowed-peer-names:
+      - "connector.aerospike.com"
+    trust-store:
+      store-type: "PEM"
+      certificate-files: "/etc/aerospike/tls/ca.pem"
+    key-store:
+      store-type: "PEM"
+      store-file: "/etc/aerospike/tls/interconnect.key.pem"
+      store-password-file: "/etc/aerospike/tls/storepass"
+      certificate-chain-files: "/etc/aerospike/tls/interconnect.crt.pem"
+    override-tls-hostname: "connector.aerospike.com"
+
+  aerospike-tls:
+    mutual-auth: false
+    trust-store:
+      store-type: "PEM"
+      certificate-files: "/etc/aerospike/tls/ca.pem"
+    key-store:
+      store-type: "PEM"
+      store-file: "/etc/aerospike/tls/aerospike.key.pem"
+      store-password-file: "/etc/aerospike/tls/storepass"
+      certificate-chain-files: "/etc/aerospike/tls/aerospike.crt.pem"
+    override-tls-hostname: "asd.aerospike.com"
+
+manage:
+  ports:
+    "5040":
+      tls-id: "service-tls"
+
+hnsw:
+  max-mem-queue-size: 4000
+  cleanup:
+    dropped-index-cleanup-scheduler-delay: 5000
+    mark-dropped-index-clean-after: 30000
+    deleted-index-retention-time: 50000
+  batch-merge:
+    parallelism: 200
+    executor-initial-delay: 600000
+  healer:
```

Comment:
There are additional healer configs such as `max-scan-rate-per-node`, `max-scan-page-size`, `cluster-max-invalid-vertices` etc. I'm not sure we want to include them here (even though its the "maximal" config), @ashishshinde WDYT?

If we would include them here it should be reflected in `1.1.0.json` file as well.

Reply:
We opted to remove the index settings from the official documentation since over riding them at the node level can be a confusing experience for the user. I'd argue we should not include them in the schema or examples following that logic.

Reply:
This whole file was generated from our schema. The schema was generated from a combination of example configs and our test config file template. I would argue that anything that is wrong or missing here needs to be added to the actual schema first.

Reply:
I think these settings just override the default (and become the new default). They are not meant to be the way to customize per index. Still though I updated to include a commented out block in the example even for maximal.